CHANGELOG/CHANGELOG-1.5.0.md

```diff
@@ -6,31 +6,50 @@
 
 ### Features 
 
+- PowerScale Replication: Implement Failback functionality. ([#558](https://github.com/dell/csm/issues/558))
+- Automated deployment of SDCs for PowerFlex on RHEL. ([#554](https://github.com/dell/csm/issues/554))
+- Update to the latest UBI/UBI Minimal images for CSM. ([#545](https://github.com/dell/csm/issues/545))
+- CSM PowerFlex : QoS parameters for throttling performance and bandwidth of a CSI driver. ([#533](https://github.com/dell/csm/issues/533))
+- Standardize csi drivers helm installation. ([#494](https://github.com/dell/csm/issues/494))
+- CSM 1.5 release specific changes. ([#491](https://github.com/dell/csm/issues/491))
+- Scheduled Backups for Application Mobility. ([#551](https://github.com/dell/csm/issues/551))
+- SLES SP4 Support for CSI PowerFlex driver in CSM. ([#539](https://github.com/dell/csm/issues/539))
+- CSI-PowerScale: Add an option to the CSI driver force the client list to be updated even if there are unresolvable host. ([#534](https://github.com/dell/csm/issues/534))
+- FC connectivity with virtualized environment. ([#528](https://github.com/dell/csm/issues/528))
 - CSM Operator Supports Install of Authorization Proxy Server. ([#511](https://github.com/dell/csm/issues/511))
 - CSI-PowerScale: Add support for Standalone Helm charts. ([#506](https://github.com/dell/csm/issues/506))
+- CSM Operator: Support installation of PowerScale metrics for Observability module. ([#505](https://github.com/dell/csm/issues/505))
```

Comment:
Please remove #505, which is a duplicate to #488.

Reply:
This will be reflected in the next PR